### ISSUE TITLE
Parser throws error when defining a function in a multiline expression

### ISSUE THREAD

**Issue by josdejong** [josdejong/mathjs#405]
The following works fine:

```js
var scope = {};
math.eval('f(x) = x^2', scope);
scope.f(3); // 9
```

but when having a multiline expression containing a function definition, we get an error:
```js
var scope = {};
math.eval('a=2; f(x) = x^a', scope);  
    // Uncaught SyntaxError: missing ) after argument list
```

**Comment by josdejong** [comment]
This has been fixed in v1.7.1

**Comment by FSMaxB** [comment]
Thanks
